README.md

```diff
@@ -80,6 +82,48 @@ Now run `yarn dev` one more time, it should be ok.
 
 The files are saved in the root directory of your cozy by default.
 
+### Use the konnector with cozy-collect
+
+To setup and run a konnector, Cozy provides a web application called `cozy-collect`.
+In order to add this konnector to the konnector's list, you'll have to edit the `config/konnectors.json` file located in `cozy-collect/src/` with the following:
+```json
+{
+  "oauth": true,
+  "slug": "google",
+  "name": "Google",
+  "editor": "Cozy Cloud",
+  "vendorLink": "google.com",
+  "categories": ["other"],
+  "fields": {
+    "access_token": {
+      "type": "hidden"
+    },
+    "refresh_token": {
+      "type": "hidden"
+    }
+  },
+  "dataType": ["contact"],
+  "source": "git://github.com/konnectors/cozy-konnector-google.git#latest",
+  "oauth_scope": "https://www.googleapis.com/auth/contacts.readonly"
+},
+```
+
+Then you'll have to register the konnector's oauth system into the cozy-stack's server with the following:
+
+```
+curl -X PUT 'localhost:5984/secrets%2Fio-cozy-account_types'
+curl -X PUT localhost:5984/secrets%2Fio-cozy-account_types/google -d '{ "grant_mode": "authorization_code", "client_id": "<CLIENT_ID>", "client_secret": "<CLIENT_SECRET>", "auth_endpoint": "https://accounts.google.com/o/oauth2/v2/auth?access_type=offline" }'
```

Comment:
You need a token endpoint, see { "grant_mode": "authorization_code", "client_id": "<CLIENT_ID>", "client_secret": "<CLIENT_SECRET>", "auth_endpoint": "https://accounts.google.com/o/oauth2/v2/auth?access_type=offline" , "token_endpoint":"https://www.googleapis.com/oauth2/v4/token"}